diff --git a/tpcds-gen/Makefile b/tpcds-gen/Makefile
index f61c2b2..bc8cb7d 100644
--- a/tpcds-gen/Makefile
+++ b/tpcds-gen/Makefile
@@ -9,8 +9,8 @@ target/tpcds_kit.zip: tpcds_kit.zip
 	cp tpcds_kit.zip target/tpcds_kit.zip
 
 tpcds_kit.zip:
-	curl http://dev.hortonworks.com.s3.amazonaws.com/hive-testbench/tpcds/README
-	curl --output tpcds_kit.zip http://dev.hortonworks.com.s3.amazonaws.com/hive-testbench/tpcds/TPCDS_Tools.zip
+#	curl http://dev.hortonworks.com.s3.amazonaws.com/hive-testbench/tpcds/README
+#	curl --output tpcds_kit.zip http://dev.hortonworks.com.s3.amazonaws.com/hive-testbench/tpcds/TPCDS_Tools.zip
 
 target/lib/dsdgen.jar: target/tools/dsdgen
 	cd target/; mkdir -p lib/; ( jar cvf lib/dsdgen.jar tools/ || gjar cvf lib/dsdgen.jar tools/ )
